jms/src/main/ballerina/Ballerina.toml

```diff
@@ -0,0 +1,3 @@
+[project]
+org-name = "ballerinax"
+version = "0.0.0"
```

Comment:
Shall we change the connector version 0.6.0 as it is the planned version during move?